src/ios/TLRKFeedback.m

```diff
@@ -34,6 +39,32 @@ -(void)initialize: (CDVInvokedUrlCommand *)command
         uid = objc_msgSend(objc_msgSend(UIDevice.currentDevice, identifierForVendorSEL), @selector(UUIDString));
     }
 
+   	// iOS localization specific options
+    // using UTF8 strings to allow displaying Chinese, Cyrillic, etc. chars 
+    NSDictionary *feedbackOptions = command.arguments.count > 2 ? command.arguments[2] : nil;
+
+    NSDictionary *iOSLocalizationOptions = [feedbackOptions valueForKey:kIOSFeedbackOptionsKeyName];
+	if(iOSLocalizationOptions != nil) {
```

Comment:
@e2l3n Here I need to check whether the _kIOSFeedbackOptionsKeyName_ exists in the options dictionary. That is why I am checking if I am not getting _nil_ from the `valueForKey`. The received options should look like:
{
enableShake: true,
iOS: {
   // other keys here
},
// other
}
This said, we need to proceed with parsing the options only if the _kIOSFeedbackOptionsKeyName_ exists.